.github/actions/install/cmake/action.yml

```diff
@@ -0,0 +1,39 @@
+name: Install CMake
+description: Download, Build and Cache CMake
+inputs:
+  version:
+    description: The desired CMake version to install
+    required: true
+  url:
```

Comment:
Is there a reason why we can't build the url from the version string ?
CMake's download urls seem to be fairly consistent and it would allow us to remove the url parameter or at least make it optional.

Reply:
It had the version twice in two different formats and I didn't take the time to parse and reformat it but it should be doable

Reply:
Ahh ok, don't bother, just wondered if we could merge them if they are always the same, but if they can differ it makes sense to keep both.

Reply:
It's 1 version just written two different way :) 
The URL has both `3.14` and `3.14.7` so it needs work getting ride of the patch version. 

We shouldn't be compiling from sources often so didn't seem worth it